drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -44,12 +45,9 @@
 
 /** Draggable Items **/
 .xblock--drag-and-drop .drag-container .items {
-    width: 210px;
-    margin: 10px;
+    display: block;
     padding: 0 !important; /* LMS tries to override this */
     position: relative;
-    display: inline;
-    float: left;
     list-style-type: none;
```

Comment:
Is this rule doing anything? AFAIK, `.items` is a `DIV` so this would have no effect.